components/Timeline/TimelineEvent/TimelineEvent.js

```diff
@@ -8,8 +8,8 @@ TimelineEvent.propTypes = {
 
 function TimelineEvent({ content, title }) {
   return (
-    <div className={styles.eventContainer}>
-      <h4 className={styles.eventTitle}>{title}</h4>
+    <div className="ml-8 mr-0 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
+      <h4 className="uppercase md:text-2xl">{title}</h4>
 
       <div className={styles.eventContent}>{content}</div>
```

Comment:
what about this className? I did not find it in the css file. @kylemh

Reply:
huh - i guess there are no styles then 🤷🏼